packages/zowe-explorer/src/uss/USSTree.ts

```diff
@@ -622,7 +622,7 @@ export class USSTree extends ZoweTreeProvider implements IZoweTree<IZoweUSSTreeN
                 }
             }
             // Sanitization: Replace multiple preceding forward slashes with just one forward slash
-            const sanitizedPath = remotepath.replace(/\/\/+/, "/");
+            const sanitizedPath = remotepath.replace(/\/\/+/, "/").replace(/\/+$/, "");
```

Comment:
Maybe we can even remove 'duplicated' filters from already existed filters?

I mean, now, once I tried two practically the same filters: `/u/users` and `/u/users/`, they both appears in my quickPick list.

<img width="1648" alt="Screenshot 2021-05-12 at 14 11 43" src="https://user-images.githubusercontent.com/74672982/117973069-37247080-b32c-11eb-9167-378473b363ba.png">

I see, that we do the same thing for filters with a lot of trailing slashes, so for example, if I will use filter: `/u/users////` and `/u/users`, there will be only one filter saved and showed in quickPick. Maybe we can do the same here?

What do you think?

Reply:
@katelynienaber Was this captured or will this be a different issue that we should open and implement in the future?

Reply:
I haven't fixed it yet but I will throw it in there because it seems small.

Reply:
@nickImbirev @jellypuno Please check this again...

Now, any duplicate slashes in the path are reduced to 1 (except at the end of the path, where they are removed).

I didn't write any code to remove previously-used filters, if they are duplicates in the list except for the number of slashes. There is a max limit on filters in the list, so as people use more, the old ones will be removed.

But I also made a global variable for the default max number of filters and whatnot. So now you can check those numbers in the globals file.